app/components/welcome.jsx

```diff
@@ -0,0 +1,25 @@
+import '../scss/hello-react';
+import React from 'react';
+import { Router, Route, Link } from 'react-router'
```

Comment:
missing semicolon, also i guess that you import this because <link > can you write a comment explaining that, please?